projects/igniteui-angular/src/lib/services/overlay/overlay.ts

```diff
@@ -304,6 +314,8 @@ export class IgxOverlayService implements OnDestroy {
                 true);
             info.settings.scrollStrategy.initialize(this._document, this, info.id);
             info.settings.scrollStrategy.attach();
+
+            this.onPositioned.emit({ id: info.id, componentRef: info.componentRef });
```

Comment:
Umm if this is `onPositioned` it would also make sense for it to emit on `reposition()`?
That might defeat the goal of the event to use as 'before show' but after 'opening/move' though.

Reply:
Yes it does!
Alternativelly, we can rename this one to onInitiallyPositioned. However this may suggest the need of an non-initial event as well.

Reply:
Whelp it feels like it overlaps way too much with the positioning strategies. `onInitial` or emit argument also seem odd, we can just name after what it's used for - content move or something